recipes/lzma_sdk/9.20/test_package/conanfile.py

```diff
@@ -1,8 +1,10 @@
-from conans import ConanFile
+from conans import ConanFile, tools
 
 
 class TestPackage(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
 
     def test(self):
-        self.run("7zDec")
-        self.run("lzma")
+        if not tools.cross_building(self.settings):
+            self.run("7zDec")
+            self.run("lzma")
```

Comment:
```suggestion
            self.run("7zDec", run_environment=True)
            self.run("lzma", run_environment=True)
```

Reply:
Forcing the package folder in PATH to avoid calling from your system.

Reply:
Ah, so it gets a somewhat pure environment; that's good. I've added it and rebased on master.